### DISCUSSION TITLE
How to remove controller from controlManager

### DISCUSSION THREAD

**Discussion by jnan806** [kubernetes-sigs/kubebuilder#3061] [Support] [answered]
we can init a controller by `NewControllerManagedBy(mgr).For(&xxObject{}).Complete(r)`, but how to shutdown the controller or remove the controller or unregister the reconciler form controlManager.

**Comment by camilamacedo86** [comment, marked as answer]
It is not possible currently. 
However, controller-runtime has an issue open with this request, see; https://github.com/kubernetes-sigs/controller-runtime/issues/1884

**Comment by camilamacedo86** [comment]
Closing since it is answered.
